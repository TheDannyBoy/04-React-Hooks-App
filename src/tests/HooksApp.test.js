import { shallow } from "enzyme"
import { HooksApp } from "../HooksApp"

describe('Testing HooksApp', () => {
  test('Should create component and match snapshot', () => {
    const wrapper = shallow(<HooksApp />);
    const h1Text = wrapper.find('h1').text();

    expect(wrapper).toMatchSnapshot();
    expect(h1Text).toBe('Hola Mundo');
  });
});
