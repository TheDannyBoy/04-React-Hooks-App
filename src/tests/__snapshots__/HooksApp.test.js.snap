// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HooksApp Should create component and match snapshot 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
